src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php

```diff
@@ -291,6 +292,12 @@ protected function setClaimRequest(ClaimRequest $claimRequest, int $empNumber):
      *         @OA\Schema(type="DateTime")
      *     ),
      *     @OA\Parameter(
+     *         name="includeEmployees",
+     *         in="query",
+     *         required=false,
+     *         @OA\Schema(type="string")
```

Comment:
```suggestion
     *         @OA\Schema(type="string", enum=ClaimRequestSearchFilterParams::INCLUDE_EMPLOYEES)
```
OR
```suggestion
     *         @OA\Schema(type="string", enum={"onlyCurrent", "onlyPast", "currentAndPast"})
```

Double check and correct https://github.com/search?q=repo%3Aorangehrm%2Forangehrm%20name%3D%22includeEmployees%22%2C&type=code these places also